src/main/kotlin/study/step4/model/Car.kt

```diff
@@ -0,0 +1,41 @@
+package study.step4.model
+
+import study.step4.util.DigitGenerator
+import study.step4.util.RandomDigit
+
+private const val ADVANCE_BASE_DIGIT = 4
+
+class Car(
+    val name: String = "",
+    private var location: Int = 0,
+    private var racingTimes: Int = 0,
+    private val digitGenerator: DigitGenerator = RandomDigit()
+) {
+    private val history = mutableListOf<Int>()
+
+    fun move() {
+        if (digitGenerator.getDigit() >= ADVANCE_BASE_DIGIT) {
+            advance()
+        } else {
+            stop()
+        }
```

Comment:
객체지향 생활 체조에 따르면, else 예약어를 지양하는것이 좋을때도 있어요!
kotlin의 경우 표현식으로 사용되는 경우도 많아, 꼭 else가 좋지않은건 아니지만,
불필요한 else문에 사용하지 않아보셔도 좋을거같아요!

stop이라는 함수는 사용하지 않아도 될거같아요!
```suggestion
        }
        saveHistory()
```

Reply:
네, 불필요한 stop 를 제거하겠습니다